src/components/chart/state-history-chart-timeline.ts

```diff
@@ -17,15 +17,21 @@ import type { TimeLineData } from "./timeline-chart/const";
  */
 const BINARY_SENSOR_DEVICE_CLASS_COLOR_INVERTED = new Set([
```

Comment:
I wonder if we don't have more inverted than not now? Should we reverse the normal logic?

Reply:
With this PR we would now have 17 of 26 device classes inverted.

=> Should I invert the inversion logic :D ?

Reply:
Yes please